### ISSUE TITLE
Double icons

### ISSUE THREAD

**Issue by atolstoy** [mauikit/buho#2]
I get weird issue with the Buho interface. All icons are painted twice:
https://i.imgur.com/GEarIOr.png
The output while Buho is running:
`atolstoy@linux-fh05:~/Sources/buho> buho
Getting collectionDB info from:  "/home/atolstoy/.local/share/buho/"
Starting DBActions
REGISTER APP "Buho" "org.kde.buho" "1.0" "Notes taking and link collector manager"
getInstance(): First instance

qrc:/src/views/links/Previewer.qml:144: TypeError: Type error
COLRO SCHEME IS VALID
COLRO SCHEME FILE EXISTS "/home/atolstoy/.local/share/color-schemes/buho.colors"
COLRO SCHEME IS VALID
COLRO SCHEME FILE EXISTS "/home/atolstoy/.local/share/color-schemes/buho.colors"
qrc:/main.qml:11:1: QML ApplicationWindow: unknown Material.primary value: 
[0918/221202.736835:WARNING:stack_trace_posix.cc(699)] Failed to open file: /home/atolstoy/#118826799 (deleted)
  Error: Нет такого файла или каталога
file:///usr/lib64/qt5/qml/org/kde/mauikit/Page.qml:122:9: QML QQuickItem: Detected anchors on an item that is managed by a layout. This is undefined behavior; use Layout.alignment instead.
qml: PREVIEW 
file:///usr/lib64/qt5/qml/org/kde/kirigami.2/GlobalDrawer.qml:517: TypeError: Cannot read property 'length' of undefined
file:///usr/lib64/qt5/qml/org/kde/kirigami.2/GlobalDrawer.qml:516: TypeError: Cannot read property 'length' of undefined
file:///usr/lib64/qt5/qml/org/kde/kirigami.2/GlobalDrawer.qml:471:53: Unable to assign [undefined] to QString
file:///usr/lib64/qt5/qml/org/kde/kirigami.2/GlobalDrawer.qml:258: TypeError: Cannot read property 'length' of undefined
file:///usr/lib64/qt5/qml/org/kde/kirigami.2/GlobalDrawer.qml:293: TypeError: Cannot read property 'length' of undefined
file:///usr/lib64/qt5/qml/org/kde/kirigami.2/GlobalDrawer.qml:292: TypeError: Cannot read property 'length' of undefined
file:///usr/lib64/qt5/qml/org/kde/mauikit/GlobalDrawer.qml:49: TypeError: Cannot read property 'width' of null
qrc:/src/widgets/NewNoteDialog.qml:23:14: Unable to assign QString to double
qrc:/src/widgets/NewNoteDialog.qml:23:14: Unable to assign QString to double
qrc:/src/widgets/NewLinkDialog.qml:23:14: Unable to assign QString to double
qrc:/src/views/links/Previewer.qml:16:14: Unable to assign QString to double
qrc:/src/widgets/CardDelegate.qml:142:31: Unable to assign QString to QQmlComponent*
qrc:/src/widgets/CardDelegate.qml:142:31: Unable to assign QString to QQmlComponent*
qrc:/src/widgets/CardDelegate.qml:119:19: Unable to assign [undefined] to QString
`

**Comment by milohr** [comment]
This is an issue with the default kde qqc2 style, with Maui style this doesn't happen.
However I will try to find a workaround.

**Comment by atolstoy** [comment]
Yes, I tried to launch it as 'buho -style=breeze', and the icons now look correct. Still, the program is hardly usable right now. The 'Add note' button (+) doesn't work every other time. It says:
qrc:/src/widgets/NewNoteDialog.qml:252: TypeError: Cannot read property 'id' of null
